docs/pages/blog/turbo-1-5-0.mdx

```diff
@@ -0,0 +1,108 @@
+Update today by running `npm install turbo@latest`.
+
+## The Monorepo Handbook
+
+Setting up a monorepo for the first time often means navigating a lot of new concepts. You'll need to understand workspaces, package installation, sharing code and dependency management - and a lot more.
+
+This often meant that folks who wanted to set up a monorepo from scratch had to piece information together from different documentation sites. First `pnpm`, then `tsup`, then back to `changesets`, then back to Turborepo for dessert.
+
+We want to fill this gap with [the Monorepo Handbook](/docs/handbook). We've built guides on how to integrate all the tools you'll need to make ship happen with your monorepo, including guides on:
+
+- [Installing Packages](/docs/handbook/package-installation)
+- [Linting](/docs/handbook/linting)
+- [Development Tasks](/docs/handbook/dev)
+- [Building Apps](/docs/handbook/building-your-app)
+- [Publishing Packages](/docs/handbook/publishing-packages)
+
+## Drop the `run`
+
+<iframe className="max-w-xl h-72 sm:h-96 w-full mt-6" src="https://www.youtube.com/embed/PEgk2v6KntY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"></iframe>
+
+You can now run tasks with the Turborepo CLI using `turbo <task>`.
+
+```diff
+- turbo run build
++ turbo build
+
+
+- turbo run lint build test
++ turbo lint build test
+```
+
+If your task name conflicts with a built-in `turbo` subcommand, we'll run our subcommand instead. That means you shouldn't name your tasks things like `prune`, `run`, or `login` - since those are built-in subcommands.
+
+`turbo run <task>` will continue to work, and there are no plans to deprecate it.
+
+## Prune now supported on `pnpm`
+
+We're delighted to announce that `turbo prune --scope=<package-name>` is now supported in `pnpm`.
+
+As part of the handbook, we've also added a section on using `turbo prune` to [build docker images](/docs/handbook/deploying-with-docker).
+
+Check out our previous [blog on prune](/blog/turbo-0-4-0#experimental-pruned-workspaces) to learn more.
+
+## Environment variables in `turbo.json`
+
+We've introduced two new keys to `turbo.json` - `env` and `globalEnv`. These allow environment variables to be configured _separately_ from tasks:
+
+```diff filename="turbo.json"
+{
+  "globalDependencies": [
+-   "$DATABASE_URL"
+  ],
++ "globalEnv": [
++   "DATABASE_URL"
++ ],
+  "pipeline": {
+    "build": {
+      "dependsOn": [
+-       "$BUILD_ENV"
+      ],
++     "env": [
++       "BUILD_ENV"
++     ]
+    }
+  }
+}
+```
+
+`globalEnv` and `env` allow you to specify a list of environment variables _without_ `$` prefixes. This makes the configuration file significantly easier to read. Read more [in our updated docs](/docs/core-concepts/caching#altering-caching-based-on-environment-variables).
+
+To help migrate from the previous syntax, we've prepared a codemod. You can run `npx @turbo/codemod env-key`.
```

Comment:
```suggestion
To help migrate from the previous syntax, we've prepared a codemod. You can run ``npx @turbo/codemod migrate-env-var-dependencies``.
```

Reply:
Yours is, admittedly better 😂 , but we've already shipped this one in 1.5 and the canary